internal/command/testversions/testprovider3

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if [ $1 == "--version" ]; then
+    exit 1
+fi
+
+exit 0
```

Comment:
You might want to add a note here why it's exit 0 vs the others that exit with 1 at the bottom.